### ISSUE TITLE
Disable/Hide all icons

### ISSUE THREAD

**Issue by royfrancis** [Powerlevel9k/powerlevel9k#1394]
I realised now that the icons take up too much space. Is there some way to disable/hide all icons?

**Comment by romkatv** [comment]
Nope.

**Comment by royfrancis** [comment]
So, hiding it individually it is.

```
POWERLEVEL9K_ANACONDA_LEFT_DELIMITER=""
POWERLEVEL9K_ANACONDA_RIGHT_DELIMITER=""
POWERLEVEL9K_TIME_ICON=""
POWERLEVEL9K_LOAD_ICON=""
POWERLEVEL9K_RAM_ICON=""
POWERLEVEL9K_HOME_ICON=""
POWERLEVEL9K_HOME_SUB_ICON=""
POWERLEVEL9K_FOLDER_ICON=""
POWERLEVEL9K_PYTHON_ICON=""
```

**Comment by romkatv** [comment]
Yes, that's the only way.

There are also brutal approaches such as `icons=()` but it'll kill too much. All separators and even the trailing whitespace before cursor will be gone.

You also probably don't want to disable icons from segments that have no other content. For example, `status` and `background_jobs` can sometimes display nothing but an icon. If you disable the icon, it'll look very odd indeed.

tl;dr: What you do is the best you can do.
